helm/cluster-api-provider-aws/values.yaml

```diff
@@ -2,7 +2,7 @@ name: cluster-api-provider-aws
 infrastructure:
   image:
     name: giantswarm/cluster-api-aws-controller
-    tag: 157fc190fc977ab1be8c974e849a343b35464c2a
+    tag: 787579502a18152ab2c1f99b7b460e60c607b7ab
```

Comment:
any chance we could use some tags version like `v0.6.5` ?

Reply:
Yeah I'd see if it works we should tag it.